app/assets/javascripts/_practice_card_utilities.es6

```diff
@@ -21,4 +21,27 @@ function truncateText() {
     truncateOnArrive('.practice-card-tagline', 120);
 }
 
-$(document).on('turbolinks:load', truncateText);
+function toggleUnderlineStylingForPracticeCardHeader() {
+    const hiddenLinkSelector = '.dm-practice-link-aria-hidden';
+    const titleSelector = '.dm-practice-title';
+    const underlineClass = 'practice-title-underline';
+
+    function findTitleSelector(targetSelector) {
+        return $(targetSelector).parent().find(titleSelector);
+    }
+
+    $(document).on('mouseenter', hiddenLinkSelector, function() {
+        findTitleSelector($(this)).addClass(underlineClass);
+    });
+
+    $(document).on('mouseleave', hiddenLinkSelector, function() {
+        findTitleSelector($(this)).removeClass(underlineClass);
+    });
```

Comment:
Clever workaround for the card / title hover. But I think you can get at most of this with SCSS. Try something like: 

```
.dm-practice-card {
  &:hover {
    .height-card-mobile h3 .dm-practice-link {
         text-decoration: underline;
        text-decoration-color: color($theme-color-accent-warm);
    }
  }
}
```

Reply:
Nice! However, this won't work, because it doesn't take into account cards that don't have an image. This option worked for me:
```
.dm-practice-card {
  &:hover {
    .dm-practice-title {
      text-decoration: underline;
      text-decoration-color: color($theme-color-accent-warm);
    }
  }
}
```